ruby/rails/security/audit/dynamic-finders.yaml

```diff
@@ -0,0 +1,29 @@
+rules:
+- id: dynamic-finders 
+  metadata:
+    owasp: 'A1: Injection'
+    cwe: "CWE-89: Improper Neutralization of Special Elements used in an SQL Command"
+    source-rule-url: https://github.com/presidentbeef/brakeman/blob/main/lib/brakeman/checks/check_dynamic_finders.rb 
+    category: security
+    technology:
+    - rails
+  message: >-
+    Discovered an application that uses MySQL and find_by_* dynamic finders 
+    on potentially sensitive fields. This could lead to 0=0 comparisons where queries will always return true,
```

Comment:
How does this lead to 0=0 comparisons?

Reply:
The vulnerability is in MySQL integer conversion, which could cause "0" to match any string -- I'll write that instead, as it seems to explain the situation better